app/src/main/java/com/yacgroup/yacguide/TourbookActivity.kt

```diff
@@ -205,7 +236,7 @@ class TourbookActivity : BaseNavigationActivity() {
 
         val layout = findViewById<LinearLayout>(R.id.tableLayout)
         layout.removeAllViews()
-        (findViewById<View>(R.id.currentTextView) as TextView).text = if (_currentYear == 0) "" else _currentYear.toString()
+        (findViewById<View>(R.id.currentTextView) as TextView).text = if (_currentYear == 0) getString(R.string.undefined) else _currentYear.toString()
```

Comment:
1. The id `R.id.currentTextView` is bit confusing.
2. I prefer the previous solution, which showed no year at all in case the year is zero. I think, from the user perspective _undefined_ makes it not clearer.

Reply:
For the tourbook view maybe, but not for the NumberPicker view. There it would be odd to have an empty field for selecting undefined ascends and I tried to avoid some conversions here like `"" <--> "unbestimmt"`

Reply:
However, I've reverted to empty string in the tourbook view.